engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/exception/RestartStrategyFromConfiguration.scala

```diff
@@ -61,9 +61,15 @@ object RestartStrategyFromConfiguration {
     val strategy = config.getString(strategyPath)
     flinkConfig.setString(restartStrategyFlinkConfigPrefix, strategy)
     config.entrySet().asScala.foreach { entry =>
-      flinkConfig.setString(s"$restartStrategyFlinkConfigPrefix.$strategy.${entry.getKey}", entry.getValue.render())
+      flinkConfig.setString(s"$restartStrategyFlinkConfigPrefix.$strategy.${entry.getKey}", toString(entry.getValue))
     }
     RestartStrategies.fromConfiguration(flinkConfig).asScala
       .getOrElse(throw new IllegalArgumentException(s"Failed to find configured restart strategy: $strategy"))
   }
+
+  private def toString(configValue: ConfigValue): String =
+    if (configValue.valueType() == ConfigValueType.STRING)
```

Comment:
WDYT about `configValue.render(ConfigRenderOptions.defaults().setJson(false)` instead of this if-else. Or it won't work?

Reply:
It won't work when value in hocon config will be quoted. The result will be:
`restart-strategy.fixed-delay.delay -> ""10 ms""`